packages/api-media-karma/__tests__/specs/index.ts

```diff
@@ -49,132 +50,153 @@ let arrayBufferPromise: Promise<ArrayBufferUpload>
 
-  test('GetImageURL should return a string without transformation', async () => {
-    const imageURL = await karmaMediaAdapter.getImageURL(TEST_UPLOAD_IMAGE)
+    test('GetImageURL should return a string without transformation', async () => {
+      const imageURL = await karmaMediaAdapter.getImageURL(TEST_UPLOAD_IMAGE)
 
-    expect(imageURL).toContain(TEST_UPLOAD_IMAGE.id)
-    expect(imageURL).toContain(TEST_UPLOAD_IMAGE.filename)
-  })
+      expect(imageURL).toContain(TEST_UPLOAD_IMAGE.id)
+      expect(imageURL).toContain(TEST_UPLOAD_IMAGE.filename)
+    })
 
-  test('GetImageURL should return a string with empty transformation', async () => {
-    const imageURL = await karmaMediaAdapter.getImageURL(TEST_UPLOAD_IMAGE, {})
+    test('GetImageURL should return a string with empty transformation', async () => {
+      const imageURL = await karmaMediaAdapter.getImageURL(TEST_UPLOAD_IMAGE, {})
 
-    expect(imageURL).toContain(TEST_UPLOAD_IMAGE.id)
-    expect(imageURL).toContain(TEST_UPLOAD_IMAGE.filename)
-    expect(imageURL).not.toContain('/t/')
-  })
+      expect(imageURL).toContain(TEST_UPLOAD_IMAGE.id)
+      expect(imageURL).toContain(TEST_UPLOAD_IMAGE.filename)
+      expect(imageURL).not.toContain('/t/')
+    })
 
-  test('GetImageURL should return a string with transformation', async () => {
-    const testImageWithFocal: Image = {
-      ...TEST_UPLOAD_IMAGE,
-      focalPoint: {
-        x: 50,
-        y: 50
+    test('GetImageURL should return a string with transformation', async () => {
+      const testImageWithFocal: Image = {
+        ...TEST_UPLOAD_IMAGE,
+        focalPoint: {
+          x: 50,
+          y: 50
+        }
       }
-    }
-    const imageURL = await karmaMediaAdapter.getImageURL(testImageWithFocal, TEST_TRANSFORMATION)
-
-    expect(imageURL).toContain(TEST_UPLOAD_IMAGE.id)
-    expect(imageURL).toContain(TEST_UPLOAD_IMAGE.filename)
-    expect(imageURL).toContain('/t/')
-    expect(imageURL).toContain(`w_${TEST_TRANSFORMATION.width}`)
-    expect(imageURL).toContain(`h_${TEST_TRANSFORMATION.height}`)
-    expect(imageURL).toContain(`q_${TEST_TRANSFORMATION.quality}`)
-    expect(imageURL).toContain(`f_50.000:50.000`)
+      const imageURL = await karmaMediaAdapter.getImageURL(testImageWithFocal, TEST_TRANSFORMATION)
+
+      expect(imageURL).toContain(TEST_UPLOAD_IMAGE.id)
+      expect(imageURL).toContain(TEST_UPLOAD_IMAGE.filename)
+      expect(imageURL).toContain('/t/')
+      expect(imageURL).toContain(`w_${TEST_TRANSFORMATION.width}`)
+      expect(imageURL).toContain(`h_${TEST_TRANSFORMATION.height}`)
+      expect(imageURL).toContain(`q_${TEST_TRANSFORMATION.quality}`)
+      expect(imageURL).toContain(`f_50.000:50.000`)
+    })
+  })
+})
+
+describe('Karma Media Adapter with Internal URL', () => {
+  const karmaMediaAdapterInternalURL = new KarmaMediaAdapter(
+    new URL(TEST_URL),
+    TEST_TOKEN,
+    new URL(TEST_INTERNAL_URL)
+  )
+
+  test('should accept UploadImage', async () => {
```

Comment:
I would also test that when you delete an image the internal url is used. And also test that when you get an image that the normal url is used.